src/queries/pr-query.ts

```diff
@@ -131,6 +130,7 @@ query PR($pr_number: Int!) {
         }
 
         files(first: 100) {
+          totalCount

```

Comment:
1. Are there any differences between the two numbers?
2. If they're always going to be the same, then it's unclear to me what would be the advantage of using `totalCount`, but shouldn't this imply also an update to tests to have this field instead of `changedFiles`?
3. (I'm assuming that the `schema/*` changes are all from regeneration.)

(If they're the same, I don't have a strong opinion either way, just seems like a lot of work to change the tests...)

Reply:
> Are there any differences between the two numbers?

Not according to the documentation:
- `changedFiles` is [The number of changed files in this pull request.](https://docs.github.com/en/free-pro-team@latest/graphql/reference/objects#pullrequest:~:text=The%20number%20of%20changed%20files%20in%20this%20pull%20request.)
- `files` [Lists the files changed within this pull request.](https://docs.github.com/en/free-pro-team@latest/graphql/reference/objects#pullrequest:~:text=Lists%20the%20files%20changed%20within%20this%20pull%20request.)

> If they're always going to be the same, then it's unclear to me what would be the advantage of using `totalCount`, but shouldn't this imply also an update to tests to have this field instead of `changedFiles`?

Good point, I've updated them now :heavy_check_mark:

I prefer `totalCount` for consistency with: (no other advantage)
https://github.com/DefinitelyTyped/dt-mergebot/blob/b50b409caab3a3e4455fcf829dc288a729142d5d/src/queries/pr-query.ts#L83-L84
https://github.com/DefinitelyTyped/dt-mergebot/blob/b50b409caab3a3e4455fcf829dc288a729142d5d/src/queries/pr-query.ts#L116-L117

> (I'm assuming that the `schema/*` changes are all from regeneration.)

Yup.

> (If they're the same, I don't have a strong opinion either way, just seems like a lot of work to change the tests...)

It was okay:
```Shell
for fileName in src/_tests/fixtures/*/_response.json; do
  git show ":$fileName" |
    jq '
      .data.repository.pullRequest
      |= (
        .files.totalCount = .changedFiles
        | del(.changedFiles)
      )
    ' > "$fileName"
done

Reply:
Is this what you actually used?  jq normally adds a newline, no?

Reply:
Yes ... There's probably a better way but this is what I used to undo the newline at end of file:
```Shell
vim '+argdo set noendofline nofixendofline | write' +quit src/_tests/fixtures/*/_response.json

Reply:
I have a very useful script that combines a bunch of text files into one file, and then operations like this are much easier regexp-replace....